File: 백준/Bronze/2562. 최댓값/최댓값.c
#include <stdio.h>

int main(void) {
   int Max, input, index;

   int arr[9];
   for(int i = 0; i < 9; i++){
      scanf("%d", &input);
      arr[i] = input;
   }

   Max = 0;
   for(int i = 0; i < 9; i++){
      if(arr[i] > Max){
         Max = arr[i];
         index = i + 1;
      }
   }

   printf("%d\n%d", Max, index);
   
   return 0;
}

File: 백준/Bronze/2562. 최댓값/최댓값.py
arr = []
for _ in range(9):
  a = int(input())
  arr.append(a)

max = 0
for i in range(len(arr)):
  if (arr[i] > max):
    max = arr[i]
    index = i + 1

print(max)
print(index)